### PR TITLE
Refactor redshift module

### PR DESCRIPTION
**Pull request by m45t3r**
I am testing this for now, however it seems quite good. No more dependency in redshft-gtk, and it as far I tested it fixes the issue #482.

### CODE REVIEW

File: i3pystatus/redshift.py [hunk 1]
```diff
@@ -1,14 +1,110 @@
...
+        # Regex to parse output
+        self._regex = re.compile(r'([\w ]+): (.+)')
+
+        # Initialize state variables
+        self._inhibited = False
+        self._temperature = 0
+        self._period = 'Unknown'
+        self._location = (0.0, 0.0)
+
+        cmd = ["redshift"]
```
Comment (richese) [inline i3pystatus/redshift.py]: I think you forgot to add user provided `args` to `cmd`.

**Comment by m45t3r** [comment]
@richese I think this is mostly fault of redshift itself:

```
$ redshift -v | sleep 1 | killall redshift && echo $?
[1]    32106 terminated  redshift -v | 
       32107 done        sleep 1 | 
       32108 done        killall redshift
0
```
As you can see, it returns 0 even if it is killed by force. I can force an error if the process exited, though.

**Comment by m45t3r** [comment]
I just report an error if redshift exit for any motive, since to get the return code of the process would complicate the code a lot (I am running redshift in a separate thread, so I would need some kind of communication). And I think this is fine, since redshift shouldn't exit anyway during execution.

What do you think @richese?

**Comment by richese** [comment]
@m45t3r 
Yeah, I think that's fine. Now it gives some feedback if you try to start redshift with invalid arguments or it exits for another reason.

**Comment by m45t3r** [comment]
I am using this for 2 Days already and didn't found any other bug. I think it is ready to merge.

@enkore, what do you think?

**Comment by teto** [comment]
Could you inhibit as a default please if it's not done already ? I don't know what others think but I find it disruptive cf https://github.com/enkore/i3pystatus/issues/489

**Comment by m45t3r** [comment]
@enkore, any comments?

**Comment by enkore** [comment]
Code-wise this looks good but needs a merge of the origin/master branch or a rebase on top of it.

**Comment by m45t3r** [comment]
@enkore Rebased.

**Pull request merged**